src/app/saga.ts

```diff
@@ -290,6 +290,11 @@ export function* handleDeepLink(action: OpenDeepLink) {
       // of our own notifications for security reasons.
       const params = convertQueryToScreenParams(rawParams.query)
       navigate(params.screen as keyof StackParamList, params)
+    } else if (rawParams.path.startsWith('/share')) {
+      const inviterAddress = rawParams.path.split('/')[2]
```

Comment:
If the inviter address is always the last instead of hardcoding it to get the second could we always get the last with `rawParams.path.split('/').pop()` instead?